atc/worker/client.go

```diff
@@ -161,16 +161,18 @@ func (client *client) RunTaskStep(
 	}
 
 	// container already exited
-	exitStatusProp, err := container.Property(taskExitStatusPropertyName)
-	if err == nil {
-		logger.Info("already-exited", lager.Data{"status": exitStatusProp})
+	exitStatusProp, _ := container.Properties()
+	for prop, code := range exitStatusProp {
```

Comment:
Hey @aledeganopix4d
Similarly, does it make sense to use a key lookup rather than iterating over the map here as well?

Reply:
Yes! Of course! :man_facepalming: 
Thanks!